components/toast/Toast.js

```diff
@@ -76,40 +69,22 @@ class Toast extends PureComponent {
   };
 
   render() {
-    const { active, children, className, label, processing } = this.props;
-
-    const toast = (
-      <Transition in={active} timeout={{ enter: 0, exit: 1000 }}>
-        {state => {
-          if (state === 'exited') {
-            return null;
-          }
+    const { children, className, label, processing } = this.props;
 
-          const classNames = cx(
-            theme['toast'],
-            {
-              [theme['is-entering']]: state === 'entering',
-              [theme['is-entered']]: state === 'entered',
-              [theme['is-exiting']]: state === 'exiting',
-            },
-            className,
-          );
+    const classNames = cx(theme['toast'], className);
 
-          return (
-            <div data-teamleader-ui="toast" className={classNames}>
-              {processing && <LoadingSpinner className={theme['spinner']} color="white" />}
-              <TextBody className={theme['label']} color="white">
-                {label}
-                {children}
-              </TextBody>
-              {this.renderCustomAction() || this.renderCustomLink() || this.renderCloseButton()}
-            </div>
-          );
-        }}
-      </Transition>
+    return (
+      <div className={theme['toast-wrapper']}>
+        <div data-teamleader-ui="toast" className={classNames}>
```

Comment:
`toast-wrapper` is not used.
And FYI: we like to put the `data-teamleader-ui` attribute on the root element.